docs/Demo/Footer.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from '@deque/cauldron-react/';
+
+const Footer = props => {
+  const year = new Date().getFullYear();
+
+  return (
+    <footer className="Footer">
+      <ul className="Footer__links">
+        <li>
+          <Link href="https://www.deque.com/terms-of-use/">Terms of Use</Link>
+        </li>
+        <li>
+          <Link href="https://www.mozilla.org/en-US/MPL/2.0/">
+            MPL-2.0 License
+          </Link>
+        </li>
+      </ul>
+      <div className="Copyright">
+        Copyright © {year}{' '}
+        <Link href="https://deque.com/">Deque Systems Inc.</Link>
```

Comment:
On the main footer for deque.com, it says "Deque Systems, Inc." I'm not sure which way is right.

It seems like your way is right.
https://grammarhow.com/comma-before-or-after-inc-company-name/

Reply:
I think it depends on how the company is registered. I'm really into the details here. 😆

Reply:
I'll match the main website - that seems best.